source/parser/identifiable_parser/identifiable_parser.hpp

```diff
@@ -10,13 +10,20 @@
 namespace sim {
 
 template <typename T>
-class IdentifieableParser {
+class IdentifieableParser  {
```

Comment:
According to fact this class does not need `T` to be `Identifiable`, I think class name should be changed:
```suggestion
class Parser  {
```